package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@netzstrategen/twig-drupal-fractal-adapter",
-  "version": "2.2.1",
+  "version": "2.3.0",
```

This appears to be a straightforward bugfix to me?
```suggestion
  "version": "2.2.2",
```

---

Makes total sense to me. I'm not sure anymore why this was bumped minor instead.